src/runtime/auto-prompt-manager-test.js

```diff
@@ -1076,6 +1083,74 @@ describes.realWin('AutoPromptManager', {}, (env) => {
     expect(alternatePromptSpy).to.not.be.called;
   });
 
+  it('should log events when a large prompt overrides the miniprompt', async () => {
+    win./*OK*/ innerWidth = 500;
+    setExperiment(win, ExperimentFlags.DISABLE_DESKTOP_MINIPROMPT, true);
+
+    await autoPromptManager.showAutoPrompt({
+      autoPromptType: AutoPromptType.CONTRIBUTION,
+      alwaysShow: true,
+      displayLargePromptFn: alternatePromptSpy,
+    });
+    await eventManagerCallback({
```

Comment:
It isn't clear how eventManagerCallback is intended to verify the behavior. This test seems to pass even with the call to logEvent commented out in the implementation code.

Reply:
Updated to use a spy instead.